UI/index.html

```diff
@@ -0,0 +1,66 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width initial-scale=1">
+
+        <title>Ride-My-Way | Welcome</title>
+        <link rel="stylesheet" href="./css/style.css">
+    </head>
+    <body>
+    <header>
+      <div class="container">
+        <div id="branding">
+          <h1><span class="highlight">Ride</span> My-Way</h1>
+        </div>
+        <nav>
+          <ul>
+            <li class="current"><a href="index.html">Home</a></li>
+            <li><a href="driver-sign-in.html"><button type="submit" class="button_2">Driver Sign-in</button></a></li>
+            <li><a href="passenger-sign-in.html"><button type="submit" class="button_2">Passenger Sign-in</button></a></li>
+          </ul>
+        </nav>
+      </div>
+    </header>
+
+    <section id="showcase">
+        <div class="container">
+            <h1>Welcome to Ride-My-Way App</h1>
+            <p>Taking Your Travel to the Next Step</p>
+        </div>
+    </section>
+
+    <section id="newsletter">
+        <div class="container">
+            <h1>Subscribe To Our Newsletter</h1>
+            <form>
+                <input type="email" placeholder="Enter Email...">
+                <button type="submit" class="button_1">Subscribe</button>
+            </form>
+        </div>
+    </section>
+
+    <section id="boxes">
+        <div class="container">
+            <div class="box" id="box1">
+                  <!--<img src="./img/sign-in.jpeg">-->
+                  <h2>Driver</h2>
+                  <p>Create your rides and enable customers to find you.</p>
+                  <p>Get a ride request while relaxed on your couch</p>
+                  <a href="driver-sign-up.html"><button type="submit" class="button_1">Sign-Up</button></a>
+            </div>
+            <div class="box" id="box2">
+                  <!--<img src="./img/sign-in.jpeg">-->
```

Comment:
Leaving commented code in your modules is not recommended. It is good to check for unused code before your check-in your changes or code.